src/main/java/codeu/model/store/basic/UserStore.java

```diff
@@ -14,6 +14,7 @@
 
 package codeu.model.store.basic;
 
+import codeu.model.data.Conversation;
```

Comment:
Do we need this extra import here?

Reply:
Oops, I think Eclipse auto-added that when I copied and pasted, I'll delete that.